### ISSUE TITLE
[BUG] 'utf-8' codec can't encode character '\udcf1'

### ISSUE THREAD

**Issue by tverrbjelke** [rdiff-backup/rdiff-backup#682]
## Bug summary

Warning: unable to read default ACL from /home/.../FS-UAE/Archiv/shared/ReqTools/Catalogs/espa�ol espa�ol: 'utf-8' codec can't encode character '\udcf1' in position 78: surrogates not allowed

(and then multiple lines like this.)

## Version, Python, Operating System

I have Debian GNU/Linux 10, installed via 
`aptitude  -t buster-backports install --with-recommends rdiff-backup`

Package: rdiff-backup                    
Version: 2.0.5-1~bpo10+1+b1

```yaml
2022-03-17 11:05:54.374122 +0100  <CLIENT-3289>  Fatal Error: No arguments given
See the rdiff-backup manual page for more information.
2022-03-17 11:05:54.369770 +0100  <CLIENT-3289>  Using rdiff-backup version 2.0.5
2022-03-17 11:05:54.369852 +0100  <CLIENT-3289>         with cpython /usr/bin/python3 version 3.7.3
2022-03-17 11:05:54.374035 +0100  <CLIENT-3289>         on Linux-4.19.0-18-amd64-x86_64-with-debian-10.11, fs encoding utf-8
```

## rdiff-backup call

How did you call rdiff-backup, with which parameters:

```
rdiff-backup  --print-statistics /home /archive/[...]
```

## What happened and what did you expect?

fyi: The target is an external zfs drive, the source is ext4. I am surprised, that utf-8 makes any problems these days! Or xattr. or what ever...

Mostly I am interested in: it is only a warning. But: **Will the files on restore be OK?** I can't speak spanish/espaniol to see if data or the metadata got corrupted in any way...

So maybe you can calm me down and explain in plain english what happened and what to do for me to get a sane backup? Or if all is fine?

**Comment by ericzolf** [comment]
It's a warning and it's only regarding ACLs, the content itself isn't impacted. I'm just surprised that the ñ isn't stored as UTF-8 character, but it's the underlying ACL library which has an issue, not rdiff-backup.
